### PR TITLE
.werft/observability: Simplify deployment logic

### PR DESCRIPTION
**Pull request by ArthurSens**
Signed-off-by: ArthurSens <arthursens2005@gmail.com>

## Description

In https://github.com/gitpod-io/observability/pull/29 we made 2 changes:

* Hid tracing components behind `tracing_enabled` external variable.
* Simplified deployment logic

This commit also do the same 2 things:
* Set to true the `tracing_enabled` external variable
* Re use upstream deployment logic instead of maintaining our own.

## Related Issue(s)

Related to https://github.com/gitpod-io/observability/issues/27

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **arthursens** after the PR has been reviewed.

Associated issue: *#27*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[.werft/observability/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/.werft/observability/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ArthurSens** [comment]
/werft run with-observability withObservabilityBranch=arthursens/components-otel-collector-22

File: .werft/observability/monitoring-satellite.ts
Comment (mads-hartmann) [inline .werft/observability/monitoring-satellite.ts:68]: Don't  you need to have observability in the path or `cd` into observability first? :)

Comment (ArthurSens) [reply .werft/observability/monitoring-satellite.ts:68]: Do you believe me that I just spend like 20 mins trying to understand why this was failing?

Thanks for the pointer 😄

**Comment by ArthurSens** [comment]
/werft run with-observability withObservabilityBranch=arthursens/components-otel-collector-22

**Comment by ArthurSens** [comment]
/werft run with-observability withObservabilityBranch=arthursens/components-otel-collector-22

**Comment by ArthurSens** [comment]
/werft run with-observability withObservabilityBranch=arthursens/components-otel-collector-22

**Comment by mads-hartmann** [comment]
@ArthurSens Please re-request review once werft is happy so that the PR shows up in my GitHub "inbox" again :)

**Comment by ArthurSens** [comment]
/werft run with-observability withObservabilityBranch=arthursens/components-otel-collector-22

**Review by mads-hartmann — APPROVED** [review]
Looks good to me 👏

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: bbcd47393d1ebaf0ab611813f9cb135a3cd79300</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/6854#pullrequestreview-814112829" title="Approved">mads-hartmann</a>*

Associated issue: *#27*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[.werft/observability/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/.werft/observability/OWNERS)~~ [mads-hartmann]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
